src/test/resources/test.template.properties

```diff
@@ -40,26 +40,17 @@ test.chromedriver.path=
 test.timeout=15
 
 ###############################################################################
-# Given below are the test accounts/passwords used for testing. 
-# The values given below will work for the dev server(except one test case GodModTest).
-# To pass GodModTest, change ALL the accounts below by appending to the back of it.
-# e.g. yourGoogleId    -> yourGoogleId.JohnDoe
-#      teammates.coord -> teammates.coord.JohnDoe
-#      alice.tmms      -> alice.tmms.JohnDoe
-#      charlie.tmms    -> charlie.tmms.JohnDoe
-#      teammates.unreg -> teammates.unreg.JohnDoe
-# If you plan to run the test suite against a staging server, 
-# you need to create four Google accounts (to use as test accounts) 
-# and give their details below.
+# Given below are the test accounts/passwords used for testing.
+# If you plan to run the test suite against a staging server, you need to create an
+# additional four Google accounts (not including the admin account) to use as test
+# accounts, then replace the following with the actual login details for all five
+# accounts. For the account names, omit '@gmail.com'.
```

Comment:
- ... you need to create ~~an~~ four additional ~~four~~ Google accounts ...
- ... replace the following **fields** with the actual ...
- (line break) For the account names, omit '@gmail.com'.